src/spac/visualization.py

```diff
@@ -10,10 +10,127 @@
 from spac.utils import check_table, check_annotation, check_feature
 
 
+def visualize_2D_scatter(

```

Comment:
@fangliu117 , would you please add tests to the functionalities implemented in visualize_2D_scatter? For example, selecting labels, adding cluster center, etc.